django/cantusdb_project/main_app/views/chant.py

```diff
@@ -112,11 +113,26 @@ def get_context_data(self, **kwargs):
 
         # If chant has a cantus ID, Create table of concordances
         if chant.cantus_id:
-            response = requests.get(
-                f"https://cantusindex.org/json-con/{chant.cantus_id}",
-                timeout=5,
-            )
-            concordances = json.loads(response.text[2:])
+            try:
+                response = requests.get(
+                    f"https://cantusindex.org/json-con/{chant.cantus_id}",
+                    timeout=5,
+                )
+                concordances = json.loads(response.text[2:])
```

Comment:
Took me testing this to understand why the string slicing is happening....so maybe a good spot for a comment?

Reply:
This is done!